app/pathogen/arbovirus/dashboard/(banner)/ArboBanner.tsx

```diff
@@ -0,0 +1,37 @@
+"use client";
+
+import React, { useContext } from "react";
+import { ArboContext } from "@/contexts/arbo-context/arbo-context";
+import { Button } from "@/components/ui/button";
+import { download, generateCsv, mkConfig } from "export-to-csv";
+
+interface ArboBannerProps {
+    className?: string;
+  }
+
+export const ArboBanner = ({className}: ArboBannerProps) => {
+    const state = useContext(ArboContext);
+    
+    const downloadData = () => {
+        const csvConfig = mkConfig({
+            useKeysAsHeaders: true,
+            filename: "arbotracker_filtered_dataset",
+          });
+          let csv = generateCsv(csvConfig)(state.filteredData);
+          download(csvConfig)(csv);
```

Comment:
!!! BEHAVIOR CHANGE !!!

I want to note something important here. The old csv download button lived on the table component, and only saved the columns you were looking at. This one is not on the table. So it can't see the columns you have selected -> nor should it. It would be very confusing if a download button not on the table was effected by the table. It still does only download the portion of the data you have filtered though (which is also confusing? maybe the button should say "download currently selected data?").

There are a few things we can do about this. Right now I left the original download button is untouched until we decide what to do: we could relabel it "download table csv" or something like that and have both? Or maybe we don't care about csv column filtering and we can just delete the other tables csv download button and not care about the missing columns.

Reply:
Things I'd love from a reviewer: what should the button(s) say? And should there be button(s) or just the one on the banner?

Reply:
These are really really good points actually. I think for now I am happy to leave it as is. I will elevate the concern to the team and we can further test it with users to see what they expect. 

One super quick fix is to add a tooltip on hover of hte button explaining it will download filtered data. 

I am ok ignoring the columns tbh, it is an easy fix for whoever downloads the data. 

Another possible solution is to simply create a popup on click. This has a couple radio buttons or something like this that lets them select whether they want all data or filtered data, alternatively it could be a dropdown... much to consider.

Happy leaving it as is for now as it is not a "breaking" change and the user will still have access to all the data if the need arises.